cri/v1alpha2/cri.go

```diff
@@ -267,12 +267,19 @@ func (c *CriManager) RunPodSandbox(ctx context.Context, r *runtime.RunPodSandbox
 		}
 	}()
 
+	// applies the runtime of container specified by the caller.
+	runtimehandler, err := c.applySandboxRuntimeHandler(sandboxMeta, r.GetRuntimeHandler(), config.Annotations)
```

Comment:
It is strange to apply a value and return the new value. After apply, we could access this value by `sandboxMeta.Runtime` directly.